src/plugins/telemetry/kibana.jsonc

```diff
@@ -6,6 +6,7 @@
     "id": "telemetry",
     "server": true,
     "browser": true,
+    "enabledOnAnonymousPages": true,
```

Comment:
this allows us to send telemetry on the login screen

Reply:
I think they are safe: 

* `sendUsageFrom` can be either `server` or `browser`
* `sendUsageTo` is either `prod` or `staging`

Reply:
Ok, then I think so too yeah. I was afraid those might be urls or something